modules/bibdocfile/lib/bibdocfile_config.py

```diff
@@ -69,3 +69,12 @@
 # CFG_BIBDOCFILE_DEFAULT_ICON_SUBFORMAT -- this is the default subformat used
 # when creating new icons.
 CFG_BIBDOCFILE_DEFAULT_ICON_SUBFORMAT = "icon"
+
+# CFG_BIBDOCFILE_SUBFORMATS_TRANSLATIONS -- a list (not dictionary, because
+# we want to preserve the order) that connects the different format sizes
+# (like 'small', 'medium', etc.) with internal format sizes (like 'icon-180', 'icon-640', etc.)
+CFG_BIBDOCFILE_SUBFORMATS_TRANSLATIONS = [
```

Comment:
`CFG_BIBDOCFILE_SUBFORMATS_TRANSLATIONS` may sound too generic.  If this variable is not going to be used for any other functionality but the "allfile" exporting, then you may want to rename it in order to indicate its purpose better.  For example `CFG_BIBDOCFILE_STREAM_ARCHIVE_FORMATS` or something of the kind, which would be nicely close to the new function `stream_archive_of_latest_files()` that does the job.